website/src/pages/tutorial.md

```diff
@@ -0,0 +1,523 @@
+## Introduction
+
+In this hands-on tutorial, we will take a look at how we can use Yew to build web applications.
+**Yew** is a modern [Rust](https://www.rust-lang.org/) framework for building front-end web apps using [WebAssembly](https://webassembly.org/).
+Yew encourages a reusable, maintainable, and well-structured architecture by leveraging Rust's powerful type system.
+A large ecosystem of community-created libraries (or crates?) provide components that work out-of-the-box and
+commonly-used patterns such as state management.
+
+Using WebAssembly also allows us to take advantage of numerous crates available on [crates.io](https://crates.io)
```

Comment:
A large ecosystem of community-created libraries, known in Rust as [crates](https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html), provide components for commonly-used patterns such as state management.
[Cargo](https://doc.rust-lang.org/cargo/), the package manager for Rust, allows us to take advantage of the numerous crates available on [crates.io](https://crates.io), such as Yew.
```
A large ecosystem of community-created libraries, known in Rust as crates, provide components for commonly-used patterns such as state management.
Cargo, the package manager for Rust, allows us to take advantage of the numerous crates available on crates.io, such as Yew.
```